### DISCUSSION TITLE
Setting variables switch-control-centre variables while to setting up

### DISCUSSION THREAD

**Discussion by thunder-007** [juspay/hyperswitch#3637] [Q&A] [answered]
Hello I'm trying to setup hyper switch following this https://docs.hyperswitch.io/hyperswitch-open-source/local-setup
```
git clone --depth 1 git@github.com:juspay/hyperswitch.git
cd hyperswitch
docker compose up -d
```
Initially 4 containers were running but then ```migration_runner-1``` stopped
Now http://localhost:8080/health says the "health is good".
Then the control center
```
git clone --depth 1 git@github.com/juspay/hyperswitch-control-center.git
cd hyperswitch-control-center
npm install --force
```
step 4 says I should edit
```
apiBaseUrl = your-backend-url
sdkBaseUrl = your-sdk-url
mixpanelToken = mixpanel-token
```
what exactly are these apiBaseUrl , sdkBaseUrl and mixpanelToken. Where can I get the mixpanelToken? I understand one of them is the docker container I'm confused of the others is there somewhere I can see the project architecture clearly and understand the workflow.

**Comment by jagan-jaya** [comment]
Hi 
If you want to use all our hyperswitch services like app server, control center, web SDK , card vault. You can use our helm charts for https://github.com/juspay/hyperswitch-helm/tree/main/charts/incubator/hyperswitch-helm kubernetes deployment and https://github.com/juspay/hyperswitch-cdk for AWS deployment

**Reply by thunder-007** [reply]
thanks @jagan-jaya 
```
apiBaseUrl = your-backend-url
sdkBaseUrl = your-sdk-url
mixpanelToken = mixpanel-token
```
can I know what shall be filled here on local host

**Reply by SanchithHegde** [reply, marked as answer]
Hey @Thunder-007, since you're running the backend server locally at port `8080`, the value of `apiBaseUrl` would be `http://localhost:8080`.
For the SDK base URL, you'd have to provide the base URL at which [hyperswitch-web](https://github.com/juspay/hyperswitch-web) is running locally, which would be `http://localhost:9050` by default, I believe.

The Mixpanel token can be empty for local setup, unless you want to have events fired for actions taken within the control center, and you'd like to view these events on the Mixpanel dashboard.
